packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerForm/AddPrayerFormConnected.js

```diff
@@ -13,42 +12,37 @@ class AddPrayerFormConnected extends React.Component {
 
   render() {
     return (
-      <Query query={getUserProfile} fetchPolicy={'cache-only'}>
-        {({ data: userData }) => (
-          <Mutation
-            mutation={ADD_PRAYER}
-            update={(cache, { data: { addPrayer } }) => {
-              const { userPrayers } = cache.readQuery({
-                query: GET_USER_PRAYERS,
-              });
-              cache.writeQuery({
-                query: GET_USER_PRAYERS,
-                data: {
-                  userPrayers: userPrayers.concat([addPrayer]),
-                },
-              });
-            }}
-          >
+      <Query query={getUserProfile}>
+        {({
+          data: {
```

Comment:
Is this way of spelling out all the things coming in somehow better than just saying `userData` and using lodash's `get` to get the pieces off of it? Or is it just user preference? I don't think it's bad (I personally like things more spelled out for me). I'm really just wondering if one is better than the other.

Reply:
this is just the way to set defaults all the way down a chain of an object, essentially what lodash does. I just prefer avoiding lodash (or any external dep) if it's not needed